lib/trento_web/controllers/about_controller.ex

```diff
@@ -4,8 +4,8 @@ defmodule TrentoWeb.AboutController do
   alias Trento.Hosts
 
   @version Mix.Project.config()[:version]
-  # TODO determine Flavor
-  @flavor "Community"
+
+  @flavor Trento.Installation.flavor()
```

Comment:
it is better not to use a module attribute here but just the function call in the info function below

Reply:
module attributes are evaluated at compile time, so if you change the `Trento.Installation` module this will recompile

Reply:
Good to know, thanks!